SwinMM/Pretrain/utils/dataset_in_memory.py

```diff
@@ -0,0 +1 @@
+../../WORD/utils/dataset_in_memory.py
```

Comment:
this doesn't look like a python script file though, could you please check the filename?

Reply:
I commented out this line and the line in the file https://github.com/Zi-hao-Wei/research-contributions/blob/5e51052958d8309a9c153d480445f7bb37b1f78c/SwinMM/Pretrain/utils/view_transforms.py please feel free to create new PRs to modify them if I misunderstood... thanks!